tests/models/test_default_evaluator.py

```diff
@@ -669,7 +764,6 @@ def example_custom_metric(eval_df, given_metrics):
         expected_metrics["true_negatives"] + expected_metrics["true_positives"],
         rtol=1e-3,
     )
-
```

Comment:
Nit: avoid noop changes

Reply:
I removed the empty line because before I forgot to group these lines together. These lines tests for the same thing, and the assertions below, also follows this grouping pattern. I'll make sure to look out to avoid these next time!

Reply:
Oh got it, you wrote these 😂